cloudinit/config/cc_phone_home.py

```diff
@@ -43,7 +43,9 @@
 
 Data is sent as ``x-www-form-urlencoded`` arguments.
 
-**Example HTTP POST**::
+**Example HTTP POST**:
```

Comment:
In order to modify the global literal highlight language (yaml in our case) we have to use an explicit code-block directive.

[literal-blocks](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks)